tock/api/urls.py

```diff
@@ -9,6 +9,7 @@
 
 urlpatterns = patterns('',
     url(r'^projects.(?P<format>csv|json)$', views.ProjectList.as_view(), name='ProjectList'),
+    url(r'^projects/(?P<pk>\d+).json$',view=views.ProjectInstanceView.as_view(),name='ProjectInstanceView'),
```

Comment:
Everything looks good :+1:  I was wondering, though, we wanted to force the `.json` here. We could remove it to make it default to JSON output. @vzvenyach